README.md

```diff
@@ -169,16 +169,6 @@ polars = { git = "https://github.com/pola-rs/polars", rev = "<optional git tag>"
 
 Required Rust version `>=1.58`
```

Comment:
The required rust version is also quite a bit outdated, I think.
A snippet like this might help.

```sh
# Find required rust nightly compiler version in https://github.com/pola-rs/polars/blob/master/.github/workflows/build-test-rust.yaml.
RUST_NIGHTLY_VERSION_FOR_POLARS=$(
    curl https://raw.githubusercontent.com/pola-rs/polars/master/.github/workflows/build-test-rust.yaml  2> /dev/null \
      | awk '{ if ($0 ~ /toolchain:/) { print $NF; exit } }'
)

rustup override set "${RUST_NIGHTLY_VERSION_FOR_POLARS}"
```